### PR TITLE
WD-5018 - fix: CLI docs link

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Done

- Fix link to the Juju web CLI

## Details

Fixes: #1503 
https://warthogs.atlassian.net/browse/WD-5018

### CODE REVIEW

File: src/urls.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ const urls = {
 
 export const externalURLs = {
   new_issue: "https://github.com/canonical/juju-dashboard/issues/new",
+  cli_help: "https://juju.is/docs/olm/the-juju-web-cli",
```
Comment (huwshimi) [inline src/urls.ts:53]: I have no idea why I originally used snake case here, but would you mind updating both of these links to camel case?

**Review by huwshimi — APPROVED** [review]
Thanks @vladimir-cucu, one little improvement you could make while you're here.

File: src/urls.ts (hunk 1 above)
Comment (vladimir-cucu) [reply src/urls.ts]: Done ✅

**Comment by huwshimi** [comment]
@vladimir-cucu this might need a rebase on main to make Codecov run again.

**Pull request merged**